README.md

```diff
@@ -1,63 +1,64 @@
 
 ## Advantages of using a Version Control System 
 
-Using a source code version control system is essential in modern-day software development practices. These tools offer powerful features and advantages to teams working on a code project.
+~~Using a source code version control system is essential in modern-day software development practices. These tools offer powerful features and advantages to teams working on a code project.~~
 
-Here's a list of a few features that a version control system can offer for your project:
-- It provides the ability to supervise and protect the codebase (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control)).
-- As already mentioned, it gives developers the ability to record all changes made to the code and save the code in those particular states (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control)).
-- By using a version control tool, a developer can analyze version history, giving them the power to revisit a code in any of the previous saved states (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control))
-- Not only does it provide a feature to revisit code, but it also allows developers to roll back the code to a previous state.
+~~Here's a list of a few features that a version control system can offer for your project:~~
+- ~~It provides the ability to supervise and protect the codebase (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control)).~~
+- ~~As already mentioned, it gives developers the ability to record all changes made to the code and save the code in those particular states (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control)).~~
+- ~~By using a version control tool, a developer can analyze version history, giving them the power to revisit a code in any of the previous saved states (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control))~~
+- ~~Not only does it provide a feature to revisit code, but it also allows developers to roll back the code to a previous state.~~
 - All these features give the developers an environment to experiment and make errors without affecting the final working code.
 - All these updates and experiments can easily be reviewed when using a version control tool.
 
-A version control tool provides many more advantages and powerful features to the developer, but one of the biggest advantages of using this tool is that multiple team members can collaborate on a single project and utilize all these features simultaneously.
+A version control tool provides many more advantages and powerful features to the developer, but one of the biggest advantages of using this tool is that multiple team members can collaborate on a single project and utilize all these features simultaneously ([ref?]()).
 
```

Comment:
The use of the word 'simultaneously' here might not be quite correct. Most development work does _not_ happen simultaneously, and so I think there is little advantage in highlighting it.

That said, this section does need a reference. If you have a reference that supports the idea of simultaneous activity being important, then feel free to leave 'simultaneous' in (with the addition of the reference/citation).